crates/storage/src/codec.rs

```diff
@@ -42,3 +42,43 @@ where
         from_proto_slice(data)
     }
 }
+
```

Comment:
Always put this to separate the test from other contents of the file

```suggestion

// ----------------------------------- test ------------------------------------

```